openreview/venue/group.py

```diff
@@ -561,3 +561,9 @@ def create_paper_group(submission):
 
         if create_paper_groups:
             tools.concurrent_requests(create_paper_group, venue.get_submissions(sort='number:asc'), desc='Creating paper groups')
+
+    def set_impersonators(self, impersonators):
```

Comment:
do you mind to add a test where a PC client can impersonate another user?

Reply:
Yeah, no problem. I wanted to ask you if you think this is the right place to have this function?

Reply:
I think this place is ok for now.